bbl_screen-patch/patches/printerui/qml/x1plus/GpioActions.js

```diff
@@ -0,0 +1,192 @@
+.pragma library
+.import X1PlusNative 1.0 as JSX1PlusNative
+.import "Binding.js" as Binding
+
+var X1Plus = null;
+var _X1PlusNative = JSX1PlusNative.X1PlusNative;
+
+
+var settings_file = "/mnt/sdcard/x1plus/buttons.json"
```

Comment:
ought this be per-printer? everything else in the settings is per-printer (i.e., stored in /config)

Reply:
you're right. I was planning on modifying this, but I wasn't sure if it would be better off in /config/ or on the sd card in /x1plus/prints/{serial}/.